package.json

```diff
@@ -129,16 +129,10 @@
   "msw": {
     "workerDirectory": "public"
   },
-  "pnpm": {
-    "overrides": {
-      "react": "^18.0.0",
-      "react-dom": "^18.0.0",
-      "@storybook/addon-docs": "^7.1.0-alpha.34",
-      "@storybook/addons": "^7.1.0-alpha.34",
-      "@storybook/api": "^7.1.0-alpha.34",
-      "@storybook/components": "^7.1.0-alpha.34",
-      "@storybook/theming": "^7.1.0-alpha.34"
```

Comment:
Should we add these as resolutions as well?

Reply:
I don't see why that would be necessary. Perhaps someone can enlighten me. It seems to work without them.

Reply:
I'm also unsure why the react resolutions were there.